django_app/redbox_app/redbox_core/dash_apps/report_app.py

```diff
@@ -53,7 +63,7 @@
             id="breakdown",
             options=[
                 {"label": "route", "value": "route"},
-                {"label": "model", "value": "chat__chat_backend"},
+                {"label": "model", "value": "chat__chat_backend__name"},
```

Comment:
this is a bug fix, `chat__chat_backend` used to be a string now its a foreign key to the LLM model